lib/livebook/teams/requests.ex

```diff
@@ -261,6 +289,14 @@ defmodule Livebook.Teams.Requests do
     |> request(method: :get, url: path, params: params)
   end
 
+  defp upload(path, content, params, team) do
+    build_req()
+    |> add_team_auth(team)
+    |> Req.Request.put_header("content-length", "#{String.length(content)}")
```

Comment:
String.length counts unicode, content length is always in bytes.

```suggestion
    |> Req.Request.put_header("content-length", "#{byte_size(content)}")
```